samples/nrf9160/modem_shell/src/location/location_shell.c

```diff
@@ -252,6 +267,13 @@ void location_ctrl_event_handler(const struct location_event_data *event_data)
 			gnss_location_work_data.format = gnss_location_to_cloud_format;
 			k_work_submit_to_queue(&mosh_common_work_q, &gnss_location_work_data.work);
 		}
+
+#if defined(CONFIG_DK_LIBRARY)
+		dk_set_led_on(LOCATION_FIX_STATUS_LED);
+		k_work_init_delayable(&location_fix_led_work, location_fix_led_worker);
```

Comment:
This initialization should probably be where `location_init()` function is called.

Reply:
Works like this but yeah, better to be in _init()